### PR TITLE
[update] Marketplace apps distro updates/minor instruction updates

### PR DESCRIPTION
**Pull request by hmorris3293**
Apps Updated:
Filecloud 
Ant Media Server (both community and Enterprise) 
BeEF
Django
Mastodon 

We are combining Ant Media Server into 1 app, so a redirect will need to be put in. The redirect is **not** included in this PR)

### CODE REVIEW

File: docs/products/tools/marketplace/guides/antmediaserver/index.md [hunk 1]
```diff
@@ -38,23 +38,39 @@ If you need adaptive streaming, cluster, load balancer, and hardware encoding, c
...
+
+### Access the Ant Media Server Dashboard
+
+1. Open your web browser and navigate to `https://[domain]:5443`, where *[domain]* can be replaced with the custom domain you entered during deployment or your Compute Instance's rDNS domain (such as `192-0-2-1.ip.linodeusercontent.com`). You can also use your IPv4 address. See the [Managing IP Addresses](/docs/products/compute/compute-instances/guides/manage-ip-addresses/) guide for information on viewing IP addresses and rDNS.
+
+1. After your account has been created, the login page appears. Enter the username and password that you just created. The username and password can be obtained in the `/home/$USERNAME/.credentials` file.
+
+1. Once logged in, the Ant Media Server Dashboard appears. From here, you can manage your entire application.
+
+    ![Screenshot of the Dashboard page](ant-media-server-dashboard.png)
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/antmediaserver/index.md:67]: @hmorris3293, Screenshot doesn't appear in the preview.

File: docs/products/tools/marketplace/guides/antmediaserver/index.md [hunk 2]
```diff
@@ -38,23 +38,39 @@ If you need adaptive streaming, cluster, load balancer, and hardware encoding, c
...
+
+1. To start using the Enterprise Edition of Ant Media Server, you need to obtain a license. Visit the [Self-Hosted Solutions](https://antmedia.io/#selfhosted) page on Ant Media's website to order a license.
+
+1. Once the license key is obtained, navigate to the Settings tab within the Ant Media Dashboard, enter the License Key, and click save.
+
+    ![Screenshot of the Settings Page](ant-media-server-settings.png)
```
Comment (jacabral) [inline docs/products/tools/marketplace/guides/antmediaserver/index.md:73]: @hmorris3293, Screenshot doesn't appear in the preview.

**Review by jacabral — APPROVED** [review]
Please update the `modified` date in the front matter of these guides.

File: docs/products/tools/marketplace/guides/antmediaserver/index.md (hunk 1 above)
Comment (hmorris3293) [reply docs/products/tools/marketplace/guides/antmediaserver/index.md:67]: @jacabral I didn't touch those screenshots in my updates, the files seem to be there and labeled correctly so we don't know why they wouldn't be appearing. They are currently working on the live docs

File: docs/products/tools/marketplace/guides/antmediaserver/index.md (hunk 2 above)
Comment (hmorris3293) [reply docs/products/tools/marketplace/guides/antmediaserver/index.md:73]: @jacabral I didn't touch those screenshots in my updates, the files seem to be there and labeled correctly so we don't know why they wouldn't be appearing. They are currently working on the live docs

**Pull request merged**